src/services/datahub-client.doubles.ts

```diff
@@ -30,11 +30,13 @@ const fakeData = [
 
 export class HardcodedDatahubClient implements DatahubClient {
   public async getAccountDomains() {
+    console.log('getAccountDomains');
     await timeout(3000);
```

Comment:
maybe for local 3seconds it's a lot of time. what do you think?

Reply:
Agree, I am changing it. 

Thanks!